lightspeed/assemblies/assembly_using-code-bot-for-suggestions.adoc

```diff
@@ -12,14 +12,16 @@ ifdef::context[:parent-context: {context}]
 
 The {AnsibleCodeBot} scans existing content collections, roles, and playbooks hosted in GitHub repositories, and proactively creates pull requests whenever best practices or quality improvement recommendations are available. The bot automatically submits pull requests to the repository, which proactively alerts the repository owner to a recommended change to their content . You can configure {AnsibleCodeBot} to scan your existing Git repositories (both public and private). Your organization must have a valid {LightspeedFullName} subscription to use Ansible code bot. 
```

Comment:
The last mention of Ansible code bot in this sentence should be an attribute.

Reply:
Good catch! I've updated the instance.